python/yugabyte_db_thirdparty/builder.py

```diff
@@ -860,12 +914,10 @@ def init_linux_clang_flags(self, dep: Dependency) -> None:
         if not is_libcxx and not is_libcxxabi and not is_libcxx_with_abi:
             log("Adding special compiler/linker flags for Clang 10+ for dependencies other than "
                 "libc++")
-            self.ld_flags += ['-lc++', '-lc++abi']
-
-            self.cxx_flags = [
-                '-stdlib=libc++',
-                '-nostdinc++'
-            ] + self.cxx_flags
+            self.ld_flags += ['-stdlib=libc++', '-lc++', '-lc++abi']
+            # TODO(asrivastava): We might not need libc++ in cxxflags but removing it causes certain
+            # builds to fail.
```

Comment:
Why don't we need it? We need to tell the complier to use libc++.

Reply:
I can see that it is being ignored during compilation of some C++ files. For example:
```
[abseil (uninstrumented)] INFO: From Compiling absl/time/internal/cctz/src/time_zone_format.cc:
[abseil (uninstrumented)] clang: warning: argument unused during compilation: '-stdlib=libc++' [-Wunused-command-line-argument]
```

Is this something we should investigate?